src/commands/mod.rs

```diff
@@ -487,11 +487,21 @@ impl DaemonControl {
                 }
             }
             coins
-                .into_values()
-                .map(|c| {
+                .into_iter()
+                .map(|(op, c)| {
+                    let ancestor_info = if c.block_info.is_none() {
+                        // We include any non-change coins here as they have been selected by the caller.
+                        // If the unconfirmed coin's transaction is no longer in the mempool, keep the
```

Comment:
I suppose it could either be that it doesn't exist or that it's been confirmed but not yet in our DB? I was initially going to leave out the coin to be on the safe side, but then decided to keep it in.